### PR TITLE
Refactor volume-group-details-data-remote-storages data source and documentation

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/power/data_source_ibm_pi_volume_group.go [hunk 1]
```diff
@@ -6,74 +6,77 @@ package power
...
-				},
-				"volume_ids": {
-					Type:     schema.TypeList,
-					Computed: true,
-					Elem:     &schema.Schema{Type: schema.TypeString},
+			Attr_StatusDescriptionErrors: {
+				Computed:    true,
+				Description: "The status details of the volume group.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
```
Comment (yussufsh) [inline ibm/service/power/data_source_ibm_pi_volume_group.go:55]: Why not use the `vgStatusDescriptionErrors()` If it is used in other resource/datasource?

File: ibm/service/power/resource_ibm_pi_volume_group.go [hunk 2]
```diff
@@ -76,7 +76,7 @@ func ResourceIBMPIVolumeGroup() *schema.Resource {
 				Computed:    true,
 				Description: "Volume Group Replication Status",
 			},
-			"status_description_errors": vgStatusDescriptionErrors(),
+			//"status_description_errors": vgStatusDescriptionErrors(),
```
Comment (yussufsh) [inline ibm/service/power/resource_ibm_pi_volume_group.go:79]: Reason for the comment?

File: ibm/service/power/data_source_ibm_pi_volume_group.go (hunk 1 above)
Comment (ismirlia) [reply ibm/service/power/data_source_ibm_pi_volume_group.go:55]: The intention was to take those out, but something might have gotten muddled in the refactor.

File: ibm/service/power/resource_ibm_pi_volume_group.go (hunk 2 above)
Comment (ismirlia) [reply ibm/service/power/resource_ibm_pi_volume_group.go:79]: @Diptipowervs please remove this comment.

File: ibm/service/power/data_source_ibm_pi_volume_group.go (hunk 1 above)
Comment (ismirlia) [reply ibm/service/power/data_source_ibm_pi_volume_group.go:55]: I think the function couldn't be fully removed without making changes to the resource also.

File: ibm/service/power/resource_ibm_pi_volume_group.go (hunk 2 above)
Comment (Diptipowervs) [reply ibm/service/power/resource_ibm_pi_volume_group.go]: Removed

Comment (yussufsh) [reply ibm/service/power/resource_ibm_pi_volume_group.go]: The full line is removed now.
Don't we need `status_description_errors`?

Comment (ismirlia) [reply ibm/service/power/resource_ibm_pi_volume_group.go]: so that's still there and got unrolled into 
```
Attr_StatusDescriptionErrors: {
```

**Comment by Diptipowervs** [comment]
--- PASS: TestAccIBMPIVolumeGroupDetailsDataSource_basic (22.16s)
PASS
--- PASS: TestAccIBMPIVolumeGroupStorageDetailsDataSourceBasic (24.64s)
PASS
--- PASS: TestAccIBMPIVolumeRemoteCopyRelationshipDataSource_basics (26.00s)
PASS

**Review by michaelkad — CHANGES_REQUESTED** [review]
Could you use constant for the changes I mentioned?

File: ibm/service/power/data_source_ibm_pi_volume_group.go (hunk 1 above)
Comment (yussufsh) [reply ibm/service/power/data_source_ibm_pi_volume_group.go:55]: I am still unclear why we are not re-using `vgStatusDescriptionErrors`? Seems the schema definition is same.

**Review by michaelkad — APPROVED** [review]
Lgtm

**Review by ismirlia — APPROVED** [review]

**Pull request merged**